Lib/pdb.py

```diff
@@ -436,6 +438,19 @@ def _cmdloop(self):
             except KeyboardInterrupt:
                 self.message('--KeyboardInterrupt--')
 
+    def _validate_file_mtime(self):
+        """Check if any of the files loaded in pdb have been modified since
+        the last time we saw it. If so, give a warning."""
```

Comment:
This comment doesn't seem right. It's only checking one file, not all the files loaded in pdb.

Reply:
Yes. This should be checking the file for the current frame. That's the one normally matters anyway as most of the source display features(probably except for `source` command) will be about the current frame.

Reply:
so should the comment be fixed?

Reply:
Fixed